src/Hero.js

```diff
@@ -25,8 +25,12 @@ export default function Hero() {
             Resources, tooling, and design guidelines for building interfaces with GitHub’s design system
           </Text>
           <Text fontFamily="mono" as="p" color="blue.3" mt={5}>
-            <LinkLight fontSize={[0, 1, 2]} href="https://styleguide.github.com/primer/">
-              Style guide
+            <LinkLight fontSize={[0, 1, 2]} href="https://primer.style/team">
+              Team
```

Comment:
Also, what do you think about the _Team_ link on the navbar? Shall we leave that as is or remove it from the navbar?

Reply:
I'll remove it from the NavBar as well, that lives in [Primer Blueprints](https://github.com/primer/blueprints) so I can do that on my end!

Reply:
Thanks for pointing that out @emplums! I was wondering about this.